cmake/test.cmake

```diff
@@ -60,28 +60,6 @@ function(add_standalone_gtest testName)
   add_dependencies(${testName} googletest)
 endfunction (add_standalone_gtest)
 
-# This function is for your typical gtest-based unit tests. Use this for
-# tests you want to run under `make test`
-function(add_gtest testName)
-  set(options "")
-  set(oneValueArgs "")
-  set(multiValueArgs SRCS DEPS)
-  cmake_parse_arguments(add_gtest
-      "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
-
-  add_executable(${testName}
-                 ${add_gtest_SRCS})
-  target_include_directories(${testName} PRIVATE
-      ${gtest_INC_DIR} ${gmock_INC_DIR})
-  target_link_libraries(${testName}
-      ${gmock_MAIN_LIB} ${gtest_LIB} ${gmock_LIB} "-pthread"
-      ${add_gtest_DEPS}
-      shared-executable
-  )
-  add_dependencies(${testName} googletest)
-  add_test(${testName} ${testName})
-endfunction (add_gtest)
-
```

Comment:
Removed because add_unit_test does everything this does and then some.

Reply:
great, thank you!